3rdparty/libcxx/CMakeLists.txt

```diff
@@ -1,85 +1,75 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-# Copy headers to collector-dir, patch, compile libcxx into intermittent library
-# place install rule
-
-# Use an external project to place the headers in the collector-dir
 include (ExternalProject)
-ExternalProject_Add(oelibcxx_cxx_includes
-    DOWNLOAD_COMMAND ""
-    CONFIGURE_COMMAND ""
-    BUILD_COMMAND ${CMAKE_COMMAND} -E copy_directory
-        ${CMAKE_CURRENT_LIST_DIR}/libcxx/include ${OE_INCDIR}/openenclave/libcxx
-        COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_LIST_DIR}/libcxx/include/__config ${OE_INCDIR}/openenclave/libcxx/__config_original
-        COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_LIST_DIR}/__config ${OE_INCDIR}/openenclave/libcxx/__config
-    INSTALL_COMMAND ""
-    )
-install (DIRECTORY ${OE_INCDIR}/openenclave/libcxx DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/3rdparty)
+ExternalProject_Add(libcxx_includes
```

Comment:
I think this should remain oelibcxx_includes, just so it is obvious that it is specific to enclaves.

Reply:
Ehh but they're explicitly not the library `oelibcxx` (though they get unioned into it). I made the names libcxx/libcxxrt/libunwind consistent to the names of the directories they're in. I would rather keep the name the same as the directory.